index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bhekisile's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
```

Comment:
Good job, but here it's highly recommended to use the `header` semantic tag, which indicates the header of the page. It'll make your code much more understandable, kindly make sure to use it.

Reply:
- [ OPTIONAL] I can see that you made some changes on your design but I noticed that you still need to improve something for your project to look more professional , as required in the previous review I also highly recommended to use the header semantic tag, which indicates the header of the page :wink: